lab_deployment/lab_deployment.md

```diff
@@ -53,7 +53,7 @@ An executable JAR file will be built that is ready to be run in production on a
 To execute the JAR file run:
 
     <copy>
-    java -jar target/demo-0.1.jar
+    java -jar target/example-0.1.jar
```

Comment:
this is inconsistent with the Gradle naming, could be fixed with `<artifactId>demo</artifactId>` in pom.xml (same with the change in lab_native_image.md)